spot-ml/src/main/scala/org/apache/spot/SpotSparkLDAWrapper.scala

```diff
@@ -83,7 +100,7 @@ object SpotSparkLDAWrapper {
     //Convert to DistributedLDAModel to expose info about topic distribution
     val distLDAModel = ldaModel.asInstanceOf[DistributedLDAModel]

```

Comment:
I wonder if we can just specify DistributedLDAModel at creation time rather than cast with asInstanceOf